lib/mayaUsd/ufe/UsdRotatePivotTranslateUndoableCommand.h

```diff
@@ -60,7 +60,11 @@ class MAYAUSD_CORE_PUBLIC UsdRotatePivotTranslateUndoableCommand : public Ufe::T
     // Ufe::TranslateUndoableCommand overrides
     void undo() override;
     void redo() override;
+#if UFE_PREVIEW_VERSION_NUM >= 2025
+    bool set(double x, double y, double z) override;
+#else
     bool translate(double x, double y, double z) override;
+#endif
```

Comment:
Looking at UFE v1 I see the translate method exists. So I think you need more than just a UFE_PREVIEW_VERSION_NUM check here. Because later when we cleanup and remove these (because Maya PR has this version) we will still need an ifdef to support the UFE v1 version with translate (not set) method.

Reply:
Great work thanks. I think we should use this same pattern moving forwards. It will be a great help when we come back and cleanup this UFE preview checks.